.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,14 @@
+// https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2jsonc
+{
+  "config": {
+    "MD013": false,
+    "MD024": false,
+    "MD033": false,
+    "MD041": false,
+    "MD026": false,
+    "MD004": { "style": "dash" } // ul-style
```

Comment:
how come this isn't enforced for CHANGELOG files?

Reply:
Because I added a `<!-- markdownlint-disable MD004 -->` directive at the top of each CHANGELOG.md file.

Reply:
Nice 👍